e2e/broadcast_test.go

```diff
@@ -8,43 +8,125 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/0xPolygon/polygon-edge/chain"
 	"github.com/0xPolygon/polygon-edge/crypto"
 	"github.com/0xPolygon/polygon-edge/e2e/framework"
 	"github.com/0xPolygon/polygon-edge/helper/tests"
 	"github.com/0xPolygon/polygon-edge/types"
-	"github.com/stretchr/testify/assert"
 )
 
 func TestBroadcast(t *testing.T) {
 	// This test is not stable
 	// Opened the ticket to check + fix it
 	t.Skip()
```

Comment:
Two points that I want to make:

- This test is anyways skipped. Do you think investing some time right now to figure out which part introduces instability? Otherwise don't see the full benefit of adding new test cases.
- This is part of the IBFT e2e framework, which is probably be deprecated at some point. Maybe it would make more sense to implement the e2e test in e2e-polybft, using its polybft e2e testing framework.

WDYT?

Reply:
Agree. Since I've already updated them, do you think it makes sense to revert it back?

Reply:
Just remember to reuse this code when revamping this test on e2e polybft.